src/util/misc.js

```diff
@@ -165,9 +165,15 @@
     /**
      * Returns coordinates of points's bounding rectangle (left, top, width, height)
      * @param {Array} points 4 points array
+     * @param {Array} [transform] 6 number trasnform matrix
```

Comment:
transform (typo on the second use).  Also mark it optional?

Is this array 1x6, 6x1, 2x3, 3x2? :-)

Reply:
All the matrices are 2 rows, 3 columns and are the standard geometric affine trasformations you use in 2d graphic. the ones you can pass to ctx.transform